src/main/java/seedu/address/logic/commands/AddContactedInfoCommand.java

```diff
@@ -19,17 +19,17 @@
  */
 public class AddContactedInfoCommand extends Command {
 
-    public static final String MESSAGE_ADD_CONTACTEDINFO_SUCCESS = "Added Contacted Info to Person: %1$s";
-    public static final String MESSAGE_DUPLICATE_CONTACTED_INFO = "Contacted list already contains: %1$s";
+    public static final String MESSAGE_ADD_INTERACTION_SUCCESS = "Recorded interaction with Person: %1$s";
```

Comment:
"Recorded interaction **to** Person: %1$s"? is it easy to put name as well? but not important, since need change test cases.